modules/serverless-logic-asyncapi-publishing-event.adoc

```diff
@@ -75,23 +74,23 @@ In your workflow, invoke the `sendResume` function within an operation state as
 [source,json]
 ----
 {
-  "states": [
-    {
-      "name": "sendEvent",
-      "type": "operation",
-      "actions": [
-        {
-          "functionRef": "sendResume",
-          "arguments": {
-            "name": "Javierito"
-          }
-        }
-      ]
-    }
-  ]
+ "states": [
+   {
+     "name": "sendEvent",
+     "type": "operation",
+     "actions": [
+       {
+         "functionRef": "sendResume",
+         "arguments": {
+           "name": "John Doe"
```

Comment:
`Javierito` was changed to `John Doe` here, but the subsequent step in this procedure is still referencing `Javierito`:

<img width="661" alt="image" src="https://github.com/user-attachments/assets/193f7850-0233-4361-ac7c-eddd48176297">

The formatting in there also looks strange, FWIW (unmarked-up `{` and `}` and I could be wrong but the `\` before `localhost` gave me pause).